docs/api/app.md

```diff
@@ -1580,6 +1592,17 @@ This is the user agent that will be used when no user agent is set at the
 app has the same user agent.  Set to a custom value as early as possible
 in your app's initialization to ensure that your overridden value is used.
 
+#### `app.userAgentMetadataFallback`
+
+A [`UserAgentMetadata`](structures/user-agent-metadata.md) object.
+
+The user agent metadata Electron will use as a global fallback.
+
+This is the metadata that will be used when no user agent metadata is set at the
+`webContents` or `session` level.  It is useful for ensuring that your entire
+app has the same user agent and metadata.  Set to a custom value as early as possible
```

Comment:
question: is early as possible = can this be set before the `ready` event?

Reply:
yes

Reply:
If I want to set different user agent and metadata on separate webcontents, do I need to wait for anything?
I assume this will set the navigator.userAgentData to match the setUserAgent metadata?
We are setting the request headers sec-ch-ua-* manually to the chosen metadata but cannot override navigator.userAgentData